### ISSUE TITLE
Cannot read property 'shape' of undefined error

### ISSUE THREAD

**Issue by ggepenyan** [vonbearshark/react-donut-chart#13]
I installed, imported the package as described, but get this
![screenshot](https://user-images.githubusercontent.com/17830568/38699843-4d6ff836-3eaa-11e8-8e17-10c88fbd9018.png)

**Comment by ggepenyan** [comment]
any help please, I don't know what to do

**Comment by vonbearshark** [comment]
What version of `react` are you using?

**Comment by DaveVanRijn** [comment]
Same problem here. I'm using React V16.2 which means PropTypes has been seperated to its own package. Any chance you are going to support the newer versions of React?

**Comment by ggepenyan** [comment]
I'm using lates version
I fixed problem in npm package, but I hope it'll be fixed, because my solution is not good enough. Package needs to be updated, to be compatible with latest version of react )) Thanks for response

**Comment by ggepenyan** [comment]
OK @vonbearshark

**Comment by ggepenyan** [comment]
what I did was importing propypes and using it
I removed `_react`... from all `_react.PropTypes.`... lines everywhere

**Comment by ggepenyan** [comment]
Still I have noticed that I was not able to customize, eg. spaces between right bar items, font sizes(for example I was not able to change main percentage font size in circle). All these things I had to do in package files, which is not the best solution in my opinion ) .Maybe it have not to be written here, bu it's important I think ))

**Comment by vonbearshark** [comment]
Alright, I think I fixed everything. I just pushed to master. If someone wants to point to that for their install and let me know if it solves all of their issues, I can publish the patch.

**Comment by vonbearshark** [comment]
If you are looking for new features, open another issue so we can talk about it there. And you can always submit a pull request for them, especially if you already have the code written.

**Comment by ggepenyan** [comment]
Thanks @vonbearshark I will test everything and let you know at this weekend

**Comment by vonbearshark** [comment]
Just waiting on someone to confirm it works

**Comment by strangerqc** [comment]
I'm currently using your fix in my app and it working fine

**Comment by ggepenyan** [comment]
Sorry for late, my pc was crashed (( I cloned the repo, it worked like charm. But it would be great if package allow to customize white spaces, font sizes ) I will add an issue for that

**Comment by vonbearshark** [comment]
Sorry for the delay. Just published :1:

**Comment by strangerqc** [comment]
Thanks @vonbearshark  !

**Comment by ggepenyan** [comment]
@vonbearshark hello. I always have to copy github repo after npm install, is that an issue only for me?

**Comment by vonbearshark** [comment]
When you add it as a dependency to a new project? What error occurs?
